wiki/Getting-Started-UIKit.md

```diff
@@ -1,22 +1,20 @@
-# Interested in seeing examples of Workflow in action?
-Start by cloning the repo and checking out the 'WorkflowExample' scheme. This should give you a decent idea of how the library works.  If you want to create a new project, read on.
+# Interested in seeing examples of SwiftCurrent in action?
 
-# CocoaPods with Programmatic UIKit
+Start by cloning the repo and checking out the 'SwiftCurrentExample' scheme. This should give you a decent idea of how the library works.  If you want to create a new project, read on.
+
+# Swift Package Manager with Programmatic UIKit
 
 ## Adding the dependency
-```ruby
-pod 'DynamicWorkflow/UIKit'
-```
-Add the above line to your Podfile.
 
-For more installation instructions, see the wiki on [installation](https://github.com/wwt/Workflow/wiki/Installation).
+For instructions on SPM and other package managers, [check out our intstallation page.](https://github.com/wwt/SwiftCurrent/wiki/Installation#swift-package-manager)
 
 ## Create your view controllers
-Create two view controllers that inherit from [UIWorkflowItem<I, O>](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/UIWorkflowItem.html).
+
+Create two view controllers that inherit from [UIWorkflowItem<I, O>](https://gitcdn.link/cdn/wwt/SwiftCurrent/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/UIWorkflowItem.html).
```

Comment:
are we updating these gitcdn links now or after?

Reply:
Will update to use the GitHub Pages links